weed/s3api/s3api_object_handlers.go

```diff
@@ -261,7 +261,7 @@ func (s3a *S3ApiServer) DeleteMultipleObjectsHandler(w http.ResponseWriter, r *h
 			}
 			if auditLog != nil {
 				auditLog.Key = entryName
-				s3err.PostAccessLog(auditLog)
+				go s3err.PostAccessLog(*auditLog)
```

Comment:
This can easily cause too many go routines if anything is blocked when writing the log.

Reply:
Yes, of course, I thought about it, but it's still better than nothing to deploy in testing.
For good, you need a queue. Do you know a good example?

Reply:
you can set the async mode to true. https://github.com/fluent/fluent-logger-golang

Reply:
> you can set the async mode to true. https://github.com/fluent/fluent-logger-golang

@chrislusf fixed